plugins/woocommerce/client/blocks/assets/js/base/components/cart-checkout/totals/discount/index.tsx

```diff
@@ -119,6 +122,8 @@ const TotalsDiscount = ( {
 					: __( 'Coupons', 'woocommerce' )
 			}
 			value={ discountTotalValue ? discountTotalValue * -1 : '-' }
+			showSkeleton={ isLoading }
+			skeleton={ <Skeleton width="45px" height="1em" /> }
```

Comment:
I'm curious why you decided to pass the skeleton and showSkeleton props into the component, rather than handling at component level? I'm also wondering if this could be derived instead of being explicitly defined. e.g. if a `TotalsDiscount` component has no value, should it render a placeholder/skeleton, instead of being fed `showSkeleton`?